### DISCUSSION TITLE
Windows 7 launch problem

### DISCUSSION THREAD

**Discussion by Rorkh** [WilliamVenner/gmpublisher#4] [Help] [answered]
Hi. When I launch "gmpublisher.exe" from latest release I got "The procedure entry point SetProcessDpiAwarenessContext could not be located in the dynamic link library user32.dll".  I am using Windows 7 and I assume it is not supported by gmpublisher. Will support for Windows 7 be added?

**Comment by WilliamVenner** [comment, marked as answer]
No, Windows 7 will never be supported because gmpublisher uses Webview2 (which is Chromium-Edge under the hood.)
